pkg/odo/cli/service/delete.go

```diff
@@ -66,30 +66,24 @@ func (o *DeleteOptions) Complete(name string, cmd *cobra.Command, args []string)
 
 // Validate validates the DeleteOptions based on completed values
 func (o *DeleteOptions) Validate() (err error) {
-	svcExists, err := o.Backend.ServiceExists(o)
+	svcDefined, err := o.Backend.ServiceDefined(o)
```

Comment:
If the command is run for a old local config (non converted) component, this will result in a panic as `o.EnvSpecificInfo.GetDevfileObj()` will return nil.

Reply:
Could you show me how to get some old local config, I would like to test it locally

Reply:
We have modified `odo create` to auto convert the newly deployed components. Thus the `s2i` flag won't work. We will need to download an older version of odo to test this scenario.

But it seems that the delete implementation for service catalog currently calls `DeleteKubernetesComponentFromDevfile()` too. So I guess we can skip this for now.